engine/lib/views.php

```diff
@@ -1918,6 +1952,8 @@ function elgg_views_boot() {
 
 	elgg_register_ajax_view('languages.js');
 	
+	elgg_register_plugin_hook_handler('view', 'all', '_elgg_views_preprocess_css');
```

Comment:
Processing all views is a big red flag for me. Limited to /cache requests this could be OK.